qa/L0_parameters/parameters_test.py

```diff
@@ -41,6 +41,8 @@
 import unittest
 import queue
 from functools import partial
+import os
+TEST_HEADER = os.environ.get('TEST_HEADER')
 
 
 class InferenceParametersTest(IsolatedAsyncioTestCase):
```

Comment:
We should probably add similar tests for our C++ client libraries.

Reply:
Added a TODO: https://jirasw.nvidia.com/browse/DLIS-4673